src/components/FiltersToggleBtn/index.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import FilterIcon from "@material-ui/icons/FilterList";
+import { Tooltip, IconButton } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import PropTypes from "prop-types";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    marginLeft: "auto",
+  },
+}));
+
+const FiltersToggleBtn = props => {
```

Comment:
Minor thing, don't need/want a change, but I never get why you'd use `Btn` if you can type `Button`